GuildedRoseLLCUITests/GuildedRoseLLCUITests.swift

```diff
@@ -10,7 +10,7 @@ class GuildedRoseLLCSpec: QuickSpec {
             app = XCUIApplication()
```

Comment:
I notice that `import XCTest` is at the top of the file. Is that import needed, since you are using Quick & Nimble?

Reply:
XCTest is necessary for XCUIApplication instantiation. 
https://developer.apple.com/documentation/xctest/xcuiapplication